tests/components/sunweg/test_config_flow.py

```diff
@@ -41,34 +97,44 @@ async def test_incorrect_login(hass: HomeAssistant) -> None:
 
 
 async def test_no_plants_on_account(hass: HomeAssistant) -> None:
-    """Test registering an integration and finishing flow with an entered plant_id."""
+    """Test registering an integration with wrong auth then with no plants available."""
```

Comment:
Why do we change this test?

Reply:
Just to cover the lines 48-49 of config_flow.py
When the authentication works, I need to mock the requests that a covered on those tests.